weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/utilities.ts

```diff
@@ -286,6 +286,39 @@ export const objectVersionNiceString = (ov: ObjectVersionSchema) => {
   return result;
 };
 
+export function parseUrlPathToRefUri(
+  urlPath: string | undefined
+): string | null {
+  if (!urlPath) {
+    return null;
+  }
+
+  // Remove leading and trailing slashes
+  const cleanPath = urlPath.replace(/^\/|\/$/g, '');
+  const segments = cleanPath.split('/');
+
+  // We need at least 6 segments to be a valid object version path
+  if (segments.length < 6) {
+    return null;
+  }
+  const entity = segments[0];
+  const project = segments[1];
+  const objectId = segments[3];
+  const versionHash = segments[5];
+
+  // Construct RefUri for object version
+  return objectVersionKeyToRefUri({
+    scheme: 'weave',
+    entity,
+    project,
+    objectId,
+    versionHash,
+    path: '',
+    refExtra: '',
```

Comment:
path isn't in the peekPath passed into this function. e.g. `peekPath = "/haruka/test-weave/objects/grammar/versions/ARz6FRfbWRsPMipJTShviHZicJVQyJTtLsDXI9gSXnM"`. Is it valid to hard code `path: 'obj'`? 

Also, i think `refExtra` isn't passed in a lot of `objectVersionKeyToRefUri` call sites so I'm comfortable leaving the refExtra empty but if anyone has a suggestion,  let me know!

Reply:
path and ref extra should both be null/empty. these are old things